app/assets/javascripts/cable/subscriptions/run.js

```diff
@@ -0,0 +1,7 @@
+$(function() {
+  if (gon.run) {
```

Comment:
Be careful of multiple subscriptions on multiple page loads since turbolinks will keep the connection active across page loads.